app.py

```
@@ -0,0 +1,536 @@
+                dataset_id = kaggle_input
+
+            # Download using existing handler
+            file_path = KaggleHandler.download_dataset(dataset_id)
+
+            # Load data
+            df = read_data(file_path)
+            st.session_state.df = df
+            st.session_state.input_source = dataset_id
+            st.success(f"Successfully loaded data from '{dataset_id}'")
+
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)